ecosystem/sep-0010.md

```diff
@@ -121,6 +126,16 @@ Every other HTTP status code will be considered an error. For example:
 }
 ```
 
+### Pooled Accounts
+
+Before defining pooled accounts, we must first define custodial accounts. In the context of Stellar, custodial accounts are Stellar accounts that are held and managed by an entity that is _not_ the owner of the funds within the account. Specifically, custodial entities hold the secret key(s) of the accounts they control. These custodial entities can create individual accounts for each user, but a much more common practice is to hold the funds of many users in a single _pooled_ account.
```

Comment:
I don't think we can use the term custodial here. I believe it is a legal term, which means it could be interpreted differently in different regions of the world, and we maybe don't want to get into that in a technical specification. Can we define what we need to define here without using the term custodial?

Reply:
Is there a better term you can think of?

Reply:
Because not all custodial accounts are pooled accounts, so we need a synonym or we need to define a term here in the SEP.

Reply:
I think we can start with "Pooled accounts are..." and define what they are without referencing custodial accounts. What drives the need to discuss custodial?

Reply:
What do we gain from using the M-address format? Memos are the tried and true solution to this problem without M-addresses, and without full `M`-address support in Horizon/SDK's I don't see a reason to use them.